afs/templates/views/components/reports/scenes/name.htm

```diff
@@ -118,12 +118,11 @@ <h2 class="afs-report-section-title"><i data-bind="click: function() {toggleVisi
     <!-- Collapsible content -->
     <div data-bind="visible: visible.classifications" class="afs-report-collapsible-container pad-lft">
 
-        <!-- ko ifnot: identifiers().length -->
+        <!-- ko ifnot: type || exactMatch().length -->
         <div class="afs-nodata-note">No classification for this resource</div>
```

Comment:
good catch above!  You'll also want to change this "no classification " to "no exact match"

Reply:
I am removing this because both the type and the exact match seem to be showing "--" when there is not values?

Reply:
pretty sure this is the same issue I keep seeing re: "dummy values" for these nodes.  That causes it to look like there's a record when there actually isn't.  In any case, removal is fine for now.

Reply:
https://github.com/archesproject/arches-for-science-prj/blob/reporting_dev/afs/media/js/utils/report.js#L45
This line is useful when we have multiple values in a single line, (eg. name_content, name_type, etc) but if the node returning a single value (as in exact match or type) it can be trickier, I guess.